.CI/build_all

```diff
@@ -25,4 +25,4 @@ git ls-tree --name-only master -- . | xargs -I {} sh -c "rm -rf {} && echo Remov
 popd > /dev/null
 
 # We just want to build all of the recipes.
-conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5"
+conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
```

Comment:
No, that is fine. Thanks!

Do you think this is ready to go? Or do you want to make some more modifications?